box.json

```diff
@@ -1,13 +1,7 @@
 {
     "chmod": "0755",
-    "directories": [
-        "src",
-        "config",
-        "vendor"
-    ],
-    "files": [
-        "LICENSE.txt",
-        "README.md"
+    "directories-bin": [
+        "config"
     ],
     "git-version": "package_version",
     "main": "bin/release",
```

Comment:
since `bin/release` is configured as a bin file in `composer.json`, this config entry is unnecessary as well; the output one is however since the output name differs from the input one

Reply:
Thank you for reviewing! I've implemented your suggestions.